### PR TITLE
DCOS-12045: New funnel graphic for declined offers summary

### PR DESCRIPTION
**Pull request by jfurrow**
This PR implements the new design for the declined offers summary:

### CODE REVIEW

File: plugins/services/src/js/components/RecentOffersSummary.js [hunk 1]
```diff
@@ -10,170 +10,166 @@ const displayedResourceValues = {
...
   ports: 'Ports'
 };
 
-const getColumnClassNameFn = (classes) => {
-  return (prop, sortBy) => {
-    return classNames(classes, {
-      'active': prop === sortBy.prop
-    });
-  };
+const maxBarHeight = 200;
```
Comment (nLight) [inline plugins/services/src/js/components/RecentOffersSummary.js]: I know we haven't discussed that but I try to go `MAX_BAR_HEIGHT` for constant top level constants.

**Comment by nLight** [comment]
Is it a correct behaviour? We're requesting any role but display `role: slave_public` as rejected.

screen shot 2016-12-14 at 19 45 45

**Comment by jfurrow** [comment]
@nLight I believe this is correct, given the data we have from the API. The logic for that is as follows:
* if the received offer does not match the request (ie. requested 6 cpus, received 4), then don't display the role data
* if the received offer matches or exceeds the resource request (ie. requested 4 cpus, received 4), but the API tells us that this resource is a reason for rejection, then display the role information in the tooltip

In this case, it does look like the API shouldn't have returned `InsufficientCpus` as a reason for this offer being rejected, but it did, so we have to display it as such.

**Comment by orlandohohmeier** [comment]
@jfurrow @nLight we don't have role "globbing" meaning it's always an exact match, therefore `slave_public` is reject if you request `*`... http://mesos.apache.org/documentation/latest/roles/

File: plugins/services/src/js/components/RecentOffersSummary.js (hunk 1 above)
Comment (orlandohohmeier) [inline plugins/services/src/js/components/RecentOffersSummary.js]: 💯 let's always capitalize top level constants.

Comment (jfurrow) [inline plugins/services/src/js/components/RecentOffersSummary.js]: This makes sense! Done!

**Comment by jfurrow** [comment]
Thanks for the clarification @orlandohohmeier!

**Comment by jfurrow** [comment]
retest this please

**Comment by jfurrow** [comment]
retest this please

**Pull request merged**